zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/SchemaManager.java

```diff
@@ -123,20 +140,35 @@ private IndexSettings getIndexSettings(final String indexName) {
 
   private Map<IndexDescriptor, Collection<IndexMappingProperty>> validateIndices(
       final IndexSchemaValidator schemaValidator, final SearchEngineClient searchEngineClient) {
-    final var currentIndices =
-        searchEngineClient.getMappings(config.getIndex().getPrefix() + "*", MappingSource.INDEX);
+    if (indexDescriptors.isEmpty()) {
+      return Map.of();
+    }
+
+    final var currentIndices = searchEngineClient.getMappings(alLIndexNames(), MappingSource.INDEX);
 
     return schemaValidator.validateIndexMappings(currentIndices, indexDescriptors);
   }
 
   private Map<IndexDescriptor, Collection<IndexMappingProperty>> validateIndexTemplates(
       final IndexSchemaValidator schemaValidator, final SearchEngineClient searchEngineClient) {
-    final var currentTemplates = searchEngineClient.getMappings("*", MappingSource.INDEX_TEMPLATE);
+    if (indexTemplateDescriptors.isEmpty()) {
+      return Map.of();
+    }
+
+    final var currentTemplates =
+        searchEngineClient.getMappings(
```

Comment:
❓ Here we should also use the template descriptors no?

Reply:
template descriptors do not support a comma seperated list of values, just a singular value https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-get-template.html. 

This is as granular was we can get I think.